decidim-admin/app/controllers/decidim/admin/moderations_controller.rb

```diff
@@ -44,6 +44,20 @@ def hide
 
       private
 
+      # Overwrites the method provided by the `Decidim::NeedsPermission` concern
+      # that checks whether the user can perform the action or not.
+      #
+      # Returns false to ensure there are no false positives.
+      def allowed_to?(*)
```

Comment:
I don't understand why you're doing this here. Can you elaborate on why preventing false positives is needed?

Reply:
This method is used in the views:

```erb
<% if can?(:unreport, moderation) || allowed_to?(:unreport, :moderation) %>
```
(see `decidim-admin/app/views/decidim/admin/moderations/index.html.erb`)

Those controllers that know about permissions, they overwrite the `allowed_to?` method via the `NeedsPermission` concern. Those that don't know about it need to fallback to some value, so since it's an `or` I make the method return `false`. If it returned `true` then it would authorize users, when the `can?` might be forbidding them to perform the action.

Still, let me re-check this part manually, just in case.

Reply:
Oh, okay 🤷‍♂️ . I still don't get why the same view can be rendered from controllers that don't implement `NeedsPermission` though, but I trust you on this.

Reply:
This is solved on #3137